common/flatpak-run-wayland.c

```diff
@@ -21,6 +21,10 @@
 #include "config.h"
 #include "flatpak-run-wayland-private.h"
 
+#include <errno.h>
+#include <limits.h>
+#include <stdlib.h>
```

Comment:
These should be unnecessary now that you're using `g_ascii_string_to_unsigned()`.

Reply:
thanks for the review! removed